### ISSUE TITLE
Too many validation errors are crashing the application

### ISSUE THREAD

**Issue by Highlife1911** [morbac/xmltools#100]
If `return multiple validation errors` is checked inside the settings, a document with too many errors crashes the application. This happens with around ~20000 validation errors.

**Comment by morbac** [comment]
I added a new option in settings dialogs which allows to set a max number of displayed errors (default: 10).
It's available in [release 3.1.1.4](https://github.com/morbac/xmltools/releases/tag/3.1.1.4)
Please can you tell me if it helps with your "extreme" validation test case ?

**Comment by Highlife1911** [comment]
No, this doesn't help. Enabling mutliple error messages still crashes the application when validating an extreme document.
